.env.example

```diff
@@ -0,0 +1,9 @@
+VERSE_MASTER_NODE_URL=
+VERSE_URL=
+VERSE_READ_NODE_URL=http://localhost:8545
+BLOCK_NUMBER_CACHE_EXPIRE_SEC=
+DATASTORE=redis
+NODE_SOCKET=ws://[::]:8546
+PORT=3000
+REDIS_URI=redis://localhost:6373
```

Comment:
@Sotatek-JohnnyNguyen 
I believe the default port for Redis is 6379. Is there a specific reason it was changed to 6373?

Reply:
I used 6373 for my redis

Reply:
@Sotatek-JohnnyNguyen 
https://github.com/oasysgames/verse-proxy/blob/fc1b97ce63bcb370bffbda97419523cdb95c89c8/docs/RateLimit.md?plain=1#L9

Since the above mentions REDIS, please adjust the README as needed.

https://github.com/oasysgames/verse-proxy/blob/master/docs/RateLimit.md

I think it would be preferable to list DATASTORE and REDIS_URI side by side.

When I ran the previous version of verse-proxy, following the README instructions (npm install, npm run build, npm run start), it started up without any errors, even without setting the REDIS_URI.
So I think it might be preferable for the values to be empty (REDIS_URI=) in the example.
What do you think?

Reply:
Or I think default port
`REDIS_URI=redis://localhost:6379`
is better.

Reply:
@Sotatek-JohnnyNguyen 
Have you checked the above?

Reply:
I have, but from my perspective, I don't think we need to change anything here since the README tells what it needs to tell about the datastore config.

Reply:
Please review my commit